src/app/components/wrapper-components/demo-card-wrapper/demo-card-wrapper.component.ts

```diff
@@ -1,4 +1,4 @@
-import { AfterViewInit, ChangeDetectorRef, Component, Input, ViewChild, ViewContainerRef } from '@angular/core';
+import { AfterViewInit, ChangeDetectorRef, Component, Input, OnChanges, SimpleChanges, ViewChild, ViewContainerRef } from '@angular/core';
```

Comment:
wtf is SimpleChanges

Reply:
le paramètre que prend "ngOnChange", ça te dit quel input a changé (et comment, je suppose)